signbank/dictionary/adminviews.py

```diff
@@ -249,6 +252,69 @@ def subquery_render_to_csv_response(self, context):
 
         return response
 
+    def ready_for_validation_render_to_csv_response(self, context):
+        if not self.request.user.has_perm("dictionary.export_csv"):
+            msg = _("You do not have permissions to export to CSV.")
+            messages.error(self.request, msg)
+            raise PermissionDenied(msg)
+
+        # Create the HttpResponse object with the appropriate CSV header.
+        # It is a Python file-like object.
+        response = HttpResponse(content_type="text/csv; charset=utf-8")
+        response[
+            "Content-Disposition"
+        ] = 'attachment; filename="ready-for-validation-export.csv"'
+
+        # Set up for outputting CSV.
+        writer = csv.writer(response)
+
+        # The queryset may or may not already be filtered for the ready for validation tag.
+        # We have to make sure it is filtered by the tag, so we are filtering again
+        ready_for_validation_qs = TaggedItem.objects.get_union_by_model(
```

Comment:
**suggestion**: consider using `get_by_model` instead of `get_union_by_model`

Looking at django-tagging's source [here](https://github.com/Fantomas42/django-tagging/blob/f3622e62112c3ecc89eabb2512b1b3dd2e5e6ca0/tagging/models.py#L295), it uses a simpler query when there is only one tag provided

Additionally, your code is correct but the name `get_union_by_model` is confusing.

I went looking into the source because I wondered if "get union" was going to return the union of the existing queryset and those with the validation tag.
This function gets the _**intersection**_ of the existing queryset and those with the validation tag.

Reply:
yep you are right. 
I had only looked at the docs and decided `get_union_by_model` seemed more suitable (don't remember why) but the sourcecode says otherwise